docs/user/rstudio/index.qmd

```diff
@@ -27,20 +29,19 @@ The following documentation helps a user understand the core workflows in RStudi
 
 ## Commercial Edition
 
-Posit Workbench has several features that differentiate it from the open-source [RStudio desktop](https://www.rstudio.com/products/rstudio/) and [RStudio Server](https://www.rstudio.com/products/rstudio/#rstudio-server), which you can read about on the [Workbench comparison page](https://www.rstudio.com/products/workbench/comparison/). The main advantages of using Posit Workbench are:
+Posit Workbench has several features that differentiate it from the open-source [RStudio desktop](https://posit.co/products/open-source/rstudio/) and [RStudio Server](https://posit.co/products/open-source/rstudio-server/), which you can read about on the [Workbench comparison page](https://posit.co/products/enterprise/workbench/). The main advantages of using Posit Workbench are:
```

Comment:
Change
    `Workbench comparison page`
to
    `PositWorkbench comparison page`

Reply:
Changed